### PR TITLE
Hooli-style Popup Task

### PR DESCRIPTION
**Pull request by nallikaea**
### Hooli-style Popup

[Demo](https://nallikaea.github.io/html-css-popup/) |
[Code base](https://github.com/nallikaea/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review ☺

### CODE REVIEW

File: submissions/nallikaea/html-css-popup/css/main.css [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+}
+
+.text-nav-item{
+  color: hsl(0, 0%, 70%);
+  font-family: 'Nanum Gothic', sans-serif;
+  font-size: 16px;
+}
+
+.menu-button, .more_button{
+  display: none;
```
Comment (madmaxWMFU) [inline submissions/nallikaea/html-css-popup/css/main.css:29]: This elements did not have a focus and did not open by space.
Try `display: none;` instead use:
```
opacity: 0;
position: absolute;

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good job!
See some comments below

**Comment by lenadgit** [comment]
According to HTML5 specification single tags should be without slash ("/") in the end. Could you please fix it?

**Comment by nallikaea** [comment]
I made amendments and changed icons, check my demo, please 🙃

File: submissions/nallikaea/html-css-popup/css/main.css [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  padding: 0 0;
+  margin: 0 0;
+}
+
+body {
+background-image: url(../popup-icons/background.png);
+background-size: cover;
+}
+
+#navigation {
```
[the remark this answers is not in the document]
Comment (nallikaea) [reply submissions/nallikaea/html-css-popup/css/main.css]: Okay, but why?

File: submissions/nallikaea/html-css-popup/css/main.css [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+  position: absolute;
+  width: 320px;
+  height: 480px;
+  right: 20px;
+  top: 50px;
+}
+
+.popup img {
+  width: 48px;
+  height: 36px;
```
[the remark this answers is not in the document]
Comment (nallikaea) [reply submissions/nallikaea/html-css-popup/css/main.css]: I fixed hight of icons because I want it be equal size. So I decided to change icons at all.

**Comment by madmaxWMFU** [comment]
Great, now we can open the menu without a mouse using only the keyboard
But there is no focus on menu buttons, some more changes needed

File: submissions/nallikaea/html-css-popup/index.html [hunk 4]
```diff
@@ -17,120 +17,120 @@
         <a  class="text-nav-item" href="#">Mail</a>
       </li>
       <li id="nav-popup">
```
Comment (lenadgit) [inline submissions/nallikaea/html-css-popup/index.html:19]: The list item can"t include all markup. One item can include only text (or text and picture). Could you please fix it? 
Each item in popup should be the list item. 
Id "nav-popup" should be the menu (with structure  nav > ul > li )

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by nallikaea** [comment]
Hi. 
I made items in popup as a list items. 
And fixed focus on popup menu buttons.

**Review by madmaxWMFU — APPROVED** [review]

**Comment by nallikaea** [comment]
Good morning, so will you merge it? :) Or I need something to do more?

**Pull request merged**